docs/testrunner-toolkit/configuration.md

```diff
@@ -124,7 +124,7 @@ https://github.com/saucelabs/testrunner-toolkit/blob/master/.sauce/testcafe.yml
 </Tabs>
 
 ## Concurrency
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight> <Highlight color="#25c2a0">testcafe</Highlight> <Highlight color="#25c2a0">playwright</Highlight></small></p>
```

Comment:
This is not true anymore, we support concurrency in both docker and sauce cloud ⚠️

Reply:
I created a new PR to update it.